### ISSUE TITLE
accessibility issues on account creation page. Unlabled Checkboxes and radio button.

### ISSUE THREAD

**Issue by taylor9209** [salesagility/SuiteCRM#1633]
When On the account page, after  clicking "Create account", the form has some accessibility issues for blind people using screen-readers.

The issues are around the email section.
First is a button that just is labeled "Array.LBL_ID_FF_ADD". I assume this is just a button to add another email address, but do to the label of this button, it is unclear.
It could be fixed by giving the button a label like "Add another Email" or something similar.

Next there is a radio button that does not have a label.
I assume by reading the email sections table that the radio button is to check the primary field.
If this is the case, giving it a lable like "Make Primary email" would work just fine.

Next, there are 2 checkboxes. 
These checkboxes should be labeled to let the screen-reader know what each checkbox does.
I assume by reading the email sections table that the next checkbox after the radio button is to select the opted out option, and the last checkbox is to say it is invalid...
But, it is unclear if this is correct or not the way it currently is.

This is using SuiteCRM 7.6.4.

**Comment by shogunpol** [comment]
As additional information, i want to say that the screen reader, when it was tested, the SuiteCRM works on Mozilla Firefox, but it doesnt work on chrome.

**Comment by mjanusauskas** [comment]
Even the labels for the text input fields should be marked up to better relate to the corresponding fields so a screen reader user would hear the label spoken when tabbing through the fields without needing to switch in and out of focus mode.
